xml_inscriptions/transcribed/MI.AA.UM.KM.L.992.xml

```diff
@@ -144,7 +148,7 @@
         
         
                 <div type="commentary">
-                    <p/>
+                    <p>B for V in "serbo" line 3.</p>
```

Comment:
For reference, I would consider marking this up in the text itself using `<reg>` and `<orig>`, even if you include this comment here.